sdks/aws/speech recognizer (from wav file input)/internal/ac8be953-62dc-494e-b9a0-999ce2c65d21.json

```diff
@@ -0,0 +1,54 @@
ioSource = createReadStream(fileName, {\r\n            highWaterMark\r\n        });\r\n        audioSource.pipe(reader);\r\n\r\n        reader.on('format', async (format) => {\r\n            const {\r\n                sampleRate,\r\n                channels,\r\n                bitDepth\r\n            } = format;\r\n\r\n            if (bitDepth > 16) {\r\n                throw new Error(\"Bit depth is bigger than 16, please use less or equal than 16 bit, in your format.\")\r\n            }\r\n            const MediaSampleRateHertz = sampleRate * channels;\r\n            const LanguageCode = language;\r\n            const MediaEncoding = 'pcm';\r\n            const audioPayloadStream = new PassThrough({\r\n                highWaterMark\r\n            });\r\n\r\n            reader.pipe(audioPayloadStream);\r\n\r\n            const client = new TranscribeStreamingClient({\r\n                region,\r\n                credentials: {\r\n                    accessKeyId,\r\n                    secretAccessKey,\r\n                },\r\n            });\r\n\r\n            const params = {\r\n                LanguageCode,\r\n                MediaEncoding,\r\n                MediaSampleRateHertz,\r\n                AudioStream: (async function* () {\r\n                    for await (const chunk of audioPayloadStream) {\r\n                        yield {\r\n                            AudioEvent: {\r\n                                AudioChunk: chunk,\r\n                            },\r\n                        };\r\n                    }\r\n                })(),\r\n            };\r\n            const command = new StartStreamTranscriptionCommand(params);\r\n\r\n            const response = await client.send(command).catch(err => {\r\n                outputs['error']({\r\n                  msg: \"aws client failed\",\r\n                  error: err\r\n              }, tag);        \r\n              });\r\n\r\n            let message = {};\r\n\r\n            for await (const event of response.TranscriptResultStream) {\r\n                if (event.TranscriptEvent?.Transcript?.Results?.length > 0) {\r\n                    outputs['current'](event);\r\n                    message = event;\r\n                }\r\n            }\r\n            outputs['result'](message)\r\n\r\n        });\r\n        reader.on('error', (err) => {\r\n             outputs['error']({\r\n                  error: \"Node wav reader failed at speech recognizer\",\r\n                  details: err\r\n              }, tag);        \r\n              });\r\n        audioSource.on('error', (err) => {\r\n            outputs['error']({\r\n                error: \"Node fs createReadStream failed at speech recognizer\",\r\n                details: err\r\n            }, tag);\r\n        })\r\n    } catch (err) {\r\n         outputs['error']({\r\n                error: \"Something went wrong at speech recognizer\",\r\n                details: err\r\n            }, tag);\r\n    }\r\n}\r\nstartRequest()"
```

Comment:
There is still one wrong output type: outputs['error']({\r\n msg: \"aws client failed\",\r\n error: err\r\n }, tag);